src/Automation/Automation/Cmdlet/UpdateManagement/NewAzureAutomationSoftwareUpdateConfiguration.cs

```diff
@@ -24,47 +24,73 @@ namespace Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement
     using Microsoft.Azure.Commands.Automation.Model.UpdateManagement;
     using System.Linq;
     using Properties;
+    using System.Collections;
+    using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
 
     [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "AutomationSoftwareUpdateConfiguration",
         SupportsShouldProcess = true, DefaultParameterSetName = AutomationCmdletParameterSets.Windows)]
     [OutputType(typeof(SoftwareUpdateConfiguration))]
     public class NewAzureAutomationSoftwareUpdateConfiguration : AzureAutomationBaseCmdlet
     {
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Schedule object used for software update configuration.")]
```

Comment:
@jemex is there any reason that the existing `ValueFromPipelineByPropertyName` property is being removed from these parameters?

Reply:
@cormacpayne I removed it because in one of your feedback, you mentioned that it is ok to remove it. It looks like I misunderstood the feedback you gave earlier. I will add it back. is that why the CI build failing?